solidserver/data_source_dns_server.go

```diff
@@ -0,0 +1,235 @@
+package solidserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"log"
+	"net/url"
+	"strings"
+)
+
+func dataSourceDNSserver() *schema.Resource {
```

Comment:
Existing resources and data-sources are not using upper case, for consistency reasons, this should be changed.